### ISSUE TITLE
Unstable training when using provided config file --> Early exiting due to MPJPE error

### ISSUE THREAD

**Issue by grau4** [mkocabas/VIBE#208]
Hi all,

I'm training VIBE using the provided config.yaml file (i.e datasets, hyperparams, seeds, etc). After the 9th epoch, I get the message:

"MPJPE error is 93.05937588214874, higher than 80.0. Exiting!"

which has been tagged as "error/pa-mpjpe" on the tensorboard writer. I wonder if anyone else has experienced this issue (?). Is there a trick on how to stabilize this loss? I'm not sure of why this loss/value has been used as exiting criteria.

Thanks!
Best
(I'm running on Windows)

**Comment by AIML** [comment]
Do you load pretrained regressor
